### DISCUSSION TITLE
plugin France TV desynchronization between image and sound

### DISCUSSION THREAD

**Discussion by novazur972** [streamlink/streamlink#3437] [Q&A]
Each time I try to watch http://www.france.tv/france-2/direct.html, I have a big (about 4 or 5 secondes) delay between image and sound.
I have no problem with others streams/plugins.
I had this issue with 1.3.1-r1 then now with 2.0.0 too.

**Comment by gravyboat** [comment]
Did you try a different video player? If you're using VLC try MPV.

**Reply by gravyboat** [reply]
I just tried with MPV it seems fine. Audio and video are in sync with some dude walking up stairs.

**Reply by novazur972** [reply]
I use MPV too, I just tested with VLC and same issue.
I tried with others France TV chanels (3, 4, 5) and same problem. You can test actually with a live show on france-3.
And the delay is about 8 seconds, not 4-5.
BUT, I realize I don't always get this default. Sometimes, I don't have any delay, but not most of the time. Very strange

Or may be there is no problem with france-3.
Edit : in fact no. I sometimes get the problem with france-3 too.

**Reply by gravyboat** [reply]
Okay on france-3 I see it as well but it doesn't happen every time. I know I converted this from an issue but I can't convert back now so I've [opened a new one](https://github.com/streamlink/streamlink/issues/3438). If you can drop your logs in there via `streamlink -l debug http://www.france.tv/france-3/direct.html` that would be great.

**Reply by novazur972** [reply]
Nice thanks.

**Reply by novazur972** [reply]
This is still not resolved.
--ffmpeg-copyts doesn't change anything there anymore.
With version 5.5.1, you have to launch streamlink several times until you have the chance to come across a shot where image and sound are synchronized.
It's very frustrating, especially in the middle of Roland Garros ;)

And https://github.com/streamlink/streamlink/issues/3438 is already closed...

**Reply by bastimeyer** [reply]
This depends on HLS packaged audio, which is currently unsupported.
https://github.com/streamlink/streamlink/issues/4721
